chcemvediet/templates/main/homepage/texts/search_text.html

```diff
@@ -7,8 +7,9 @@
  %
 {% endcomment %}
 
-
+<div class="chv-bellow-1 center-block">
```

Comment:
@martinmacko47 zabalil som to cele do `<div>`, tak som dokazal zachovat rovnaky padding.

Reply:
@viliambalaz Nie, `texts/*` templaty by mali mat v sebe iba text a jednoduche tagy ako `<p>`, `<b>` a podobne. Tieto templaty su myslene na editovanie ludmi, ktori nam pisu texty a nevedia html/css. Preto v nich nema byt ziadna logika.

Presun teda `<div class="chv-bellow-1 center-block">` von z `search_text.html` do `homepage/main.html`.